### ISSUE TITLE
v0.3.1766 breaks intellisense & "go to definition" for generated tonic types

### ISSUE THREAD

**Issue by mikeparisstuff** [rust-lang/rust-analyzer#16109]
After updating to v0.3.1766 "go to definition" and intellisense breaks for tonic generated types. Reverting to version v0.3.1756 fixes the issue. I suspect it has something to do with the handling of `include!` macros as the tonic code-gen creates modules like this:

```rust
pub mod protos {
    pub mod accounts {
        pub mod service {
            pub mod v1 {
                include!("protos.accounts.service.v1.rs");
            }
        }
    }
}
```

The "go to definition" works for 

`aiproto::protos::aimodels::v1` 

but not for 

`aiproto::protos::aimodels::v1::Account` which lives behind the `include!`. 

RA does not report any other errors and running `cargo build` continues to build successfully. Let me know if you need more details.

**rust-analyzer version**: v0.3.1766

**rustc version**: rustc 1.74.1 (also tried with rustc 1.70.*)

**relevant settings**: Its specific to the v0.3.1766 version as rolling back to v0.3.1756 fixes the issue

**Comment by awgn** [comment]
I encountered the same issue as well.  I also tried the RA pre-release version 0.4.1769, but it has the same problem. 

Furthermore, for the sake of convenience, I reverted to version 0.3.1740, and it also works without any issues.
